helix-core/src/test/java/org/apache/helix/TestHelper.java

```diff
@@ -76,7 +76,7 @@
 
 public class TestHelper {
   private static final Logger LOG = LoggerFactory.getLogger(TestHelper.class);
-  public static final long WAIT_DURATION = 20 * 1000L; // 20 seconds
+  public static final long WAIT_DURATION = 60 * 1000L; // 60 seconds
```

Comment:
This reminds me that we are not testing for the latency for now. For example, if the callback does not happen within one second, then we may want to fail the test.
Will it be a good idea to have at least 2 Duration?
Pipeline_Wait_Duration = 60000;
Callback_Wait_Duration = 2000;

Reply:
In this pull request, we don't have the case of waiting too long, we should fail the test case. How about let us add them once we have this case.

Reply:
Sure. This is not a big concern.